sormas-api/src/main/java/de/symeda/sormas/api/event/EventParticipantFacade.java

```diff
@@ -66,6 +66,10 @@
 
 	List<EventParticipantDto> getAllActiveEventParticipantsAfter(Date date, Integer batchSize, String lastSynchronizedUuid);
 
+	List<String> getArchivedUuidsSince(Date since);
+
+	void archiveAllArchivableEventParticipants(int daysAfterEventParticipantGetsArchived);
```

Comment:
THis method should not be part of the facade interface (to the outside). It should just be a public method of the facadeEjb that can be called from other backend classes - in this case the CronService.
Same for ImmunizationFacade, etc.

Reply:
removed the method from interface